mlflow/store/model_registry/abstract_store.py

```diff
@@ -321,6 +323,38 @@ def get_model_version_by_alias(self, name, alias):
         """
         pass
 
+    @abstractmethod
+    def copy_model_version(self, src_mv, dst_name):
+        """
+        Copy a model version from one registered model to another as a new model version.
+
+        :param src_mv: A :py:class:`mlflow.entities.model_registry.ModelVersion` object representing
+                       the source model version.
+        :param dst_name: the name of the registered model to copy the model version to. If a
+                         registered model with this name does not exist, it will be created.
+        :return: Single :py:class:`mlflow.entities.model_registry.ModelVersion` object representing
+                 the cloned model version.
+        """
+        pass
+
+    def _copy_model_version_impl(self, src_mv, dst_name):
+        try:
+            self.create_registered_model(dst_name)
+        except MlflowException as e:
+            if e.error_code != RESOURCE_ALREADY_EXISTS:
+                raise
+
+        source_uri = f"models:/{src_mv.name}/{src_mv.version}"
+
+        return self.create_model_version(
+            name=dst_name,
+            source=source_uri,
+            run_id=src_mv.run_id,
+            tags=[ModelVersionTag(k, v) for k, v in src_mv.tags.items()],
+            run_link=src_mv.run_link,
+            description=src_mv.description,
+        )
```

Comment:
```suggestion
        return self.create_model_version(
            name=dst_name,
            source=f"models:/{src_mv.name}/{src_mv.version}",
            run_id=src_mv.run_id,
            tags=[ModelVersionTag(k, v) for k, v in src_mv.tags.items()],
            run_link=src_mv.run_link,
            description=src_mv.description,
        )
```

I'd remove `source_uri`. It seems redundant.